message_definitions/v1.0/messages.xml

```diff
@@ -2480,6 +2480,18 @@
       <field name="course"  type="float" unit="deg"/>
     </message>
 
+    <message name="GROUND_GPS" id="57" link="forwarded">
```

Comment:
I might be interested with the same kind of information for other kinds of target. What do you think about a more generic message (like `GROUND_TARGET`, `TARGET_GPS`, whatever) and add an identifier field, eventually with a reserved value like 0 for the GCS ?

Reply:
That is fine for me also ;)

Reply:
```suggestion
    <message name="TARGET_POS" id="57" link="forwarded">
```

So far GPS in message names is only used for sensors. Do you mind using POS or LOCATION.